src/Store.js

```diff
@@ -519,6 +520,16 @@ class Store {
   _addOperationBatch (data, batchOperation, lastOperation, onProgressCallback) {
     throw new Error('Not implemented!')
   }
+  
+  _procEntry(entry) {
+    var { payload, hash } = entry;
+    var { op } = payload;
+    if(op) {
+      this.events.emit(`log.op.${op}`, this.address.toString(), hash, payload);
```

Comment:
Thinking about the interface for the event here, ie. what the user subscribes to, and I feel it may be better to pass the `op` as an arg in the emitted event as opposed to being part of the event name.

The reason is that the OP codes are dependant on the specific stores, ie. in can be an infinite amount of different opcodes, and having it as part of the event name effectively means the API for the event is "infinite".

What do you think @vaultec81, would it make sense to put the `op` in the args and call the emitted event simply `log.op`?

Reply:
Thinking this a bit further, would it make sense to call this event `updated` instead of `log-op`? I feel that'd be more high-level and tangible than `log.op` as the event name.

Reply:
Yeah I agree that it should be a argument instead of part of the event. The only use case I can imagine would be if there is a very specific set of events that are of interest. This is assuming that the store implementation will document all supported operations.
I think `log.op` is an appropriate name. The event is triggered when there is a new entry added to the oplog.